tests/tests_profile.txt

```diff
@@ -1,2684 +0,0 @@
-============================= test session starts ==============================
```

Comment:
@mehdisadeghi why did this file got deleted? accident?

Reply:
@carocad actually I removed it because it sounds like unnecessary to profile tests. Instead I focused on profiling only the script (which tests will use anyway).